CMakeLists.txt

```diff
@@ -549,10 +549,6 @@ ENDIF()
 IF(WITHOUT_SERVER)
   MESSAGE(FATAL_ERROR "This branch does not support -DWITHOUT_SERVER=ON configuration.")
 ENDIF()
-# Disallow WITH_EMBEDDED_SERVER=OFF builds
-IF(NOT WITH_EMBEDDED_SERVER)
-  MESSAGE(FATAL_ERROR "This branch does not support -DWITH_EMBEDDED_SERVER=OFF configuration.")
-ENDIF()
```

Comment:
Removed by upstream or you? Any reason to remove this?

Reply:
removed by upstream